templates/taxi/car_list.html

```diff
@@ -1,17 +1,40 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1 class="text-center">Car List</h1>
+
+  <p class="text-lg-end">
+    <a class="btn btn-primary" href="{% url 'taxi:car-create' %}">Create new car</a>
+  </p>
+
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th scope="col">Id</th>
+        <th scope="col">Car Model</th>
+        <th scope="col">Manufacturer</th>
+        <th scope="col">Update</th>
+        <th scope="col">Delete</th>
+      </tr>
+    </thead>
+  <tbody>
```

Comment:
>
2. On the car list page add button that leads to the car creation page.
**On the car detail page add buttons that lead to the car update page and car deletion page.**